internal/Get-XpDirTreeRestoreFile.ps1

```diff
@@ -48,13 +48,16 @@ function Get-XpDirTreeRestoreFile {
 		$Path = $Path + "\"
 	}
 	
-	If (!(Test-DbaSqlPath -SqlInstance $server -path $path)) {
+	If (!(Test-DbaSqlPath -SqlInstance $Server -path $path)) {
 		Stop-Function -Message "SqlInstance $SqlInstance cannot access $path" -EnableException $true
 	}
-	
-	$query = "EXEC master.sys.xp_dirtree '$Path',1,1;"
-	$queryResult = Invoke-Sqlcmd2 -ServerInstance $SqlInstance -Credential $SqlCredential -Database tempdb -Query $query
-	#$queryResult = $Server.Query($query)
+	if ($Server.VersionMajor -lt 9) {
+		$query = "EXEC master..xp_dirtree '$Path',1,1;"
+	} else {
+		$query = "EXEC master.sys.xp_dirtree '$Path',1,1;"
+	}
+	#$queryResult = Invoke-Sqlcmd2 -ServerInstance $SqlInstance -Credential $SqlCredential -Database tempdb -Query $query
+	$queryResult = $Server.Query($query)
```

Comment:
Do we want to add a `Write-Message -Level Debug` to this so we can output that query execution?